tests/migration/online_migration/pre_migration.pm

```diff
@@ -21,10 +21,17 @@ sub check_or_install_packages {
     if (get_var("FULL_UPDATE") || get_var("MINIMAL_UPDATE")) {
         # if system is fully updated or even minimal patch applied, all necessary packages for online migration should be installed
         # check if the packages was installed along with update
-        my $output = script_output "rpm -qa yast2-migration zypper-migration-plugin rollback-helper | sort";
-        if ($output !~ /rollback-helper.*?yast2-migration.*?zypper-migration-plugin/s) {
+        zypper_call('rm zypper-migration-plugin') if is_leap_migration;
+        my $output = script_output "rpm -qa yast2-migration zypper-migration-plugin rollback-helper yast2-migration-sle | sort";
+        if ($output !~ /rollback-helper.*?yast2-migration.*?zypper-migration-plugin. *?yast2-migration-sle/s) {
```

Comment:
Need we check the yast2-registration here also?

Reply:
No need and I've run regression tests for it.
https://openqa.nue.suse.com/t8580869

Reply:
No need and I've run regression tests for it.
https://openqa.nue.suse.com/t8580869